packages/logic/src/index.ts

```diff
@@ -191,6 +193,36 @@ export const setReplyMessage: Actions["setReplyMessage"] = (
   useGlobalStore.setState((state) => {
     state.EMAIL_CLIENT.COMPOSING.respondingThread = thread;
     state.EMAIL_CLIENT.COMPOSING.messageType = replyType;
+    if (replyType === "forward") {
+      const latestProviderMessageId = thread?.email_provider_message_id.at(-1);
```

Comment:
Can we wrap or compute this in the thread object itself? It’s not super readable and look’s dangerous 😱